themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Project-Scoped Stacks in Self-Managed State Backends"
+date: 2023-03-29
+meta_desc: The latest Pulumi update includes support for stack names scoped by project name in self-managed state backends.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+tags:
+    - features
+    - pulumi-releases
+---
+
+The latest Pulumi update includes support for project-scoped stacks in self-managed state backends, making self-managed state backends consistent with how stacks are scoped in the Pulumi Service. Stacks created in new or empty self-managed backends will now be scoped by project by default, and existing backends can be upgraded to be scoped by project. This allows using the same simple stack names such as "dev", "staging", "production", etc. across multiple projects within the same self-managed backend.
+
+<!--more-->
+
+## What are State Backends?
+
+Each Pulumi [stack](/docs/intro/concepts/stack/) has its own metadata about your infrastructure so it can manage your cloud resources. This metadata is called [_state_](/docs/intro/concepts/state/) and it is stored in a _backend_ of your choosing: **Service** or **Self-Managed**.
```

Comment:
Just 'cause not a proper noun:

```suggestion
Each Pulumi [stack](/docs/intro/concepts/stack/) has its own metadata about your infrastructure so it can manage your cloud resources. This metadata is called [_state_](/docs/intro/concepts/state/) and it is stored in a _backend_ of your choosing: **Service** or **self-managed**.
```